examples/reliable_conversation/test_basic.py

```
@@ -0,0 +1,340 @@
+
+import asyncio
+import sys
+import os
+from pathlib import Path
+
+# Add src to path for imports
+sys.path.insert(0, str(Path(__file__).parent / "src"))
+
+from mcp_agent.app import MCPApp
+from workflows.conversation_workflow import ConversationWorkflow
+from models.conversation_models import ConversationState, QualityMetrics, Requirement
```

Ruff (0.11.9)

`models.conversation_models.QualityMetrics` imported but unused

Remove unused import

(F401)